lib/sanitizeCurrency.js

```diff
@@ -0,0 +1,260 @@
+// Valid currencies
+const validCodes = [
+  'ADF',
```

Comment:
What is the source for this list? Could we maybe rely on third party library for this validation, to an extend.
Some of them seem to be bound to specific time periods and transition to euro: https://en.wikipedia.org/wiki/ISO_4217#Historical_currency_codes Don't know if that can be useful somehow.

Reply:
The whole list comes from https://fr.wikipedia.org/wiki/ISO_4217#Index_par_code

If we know the conversion rate from the old currency to the new currency, then we could use it during the extraction but this won't be an exact information (the conversion rate would be the one at the time of the transition)

Will ask Laurent...

Reply:
Thanks for sharing this info! Could you please add it as a comment, it'd be useful for any type of reminders in the future.